static/app/test/specs/entities.spec.js

```diff
@@ -8,27 +8,102 @@ var Radio = require('backbone.radio');
 
 // App
 var EntitiesAPI = require('app/entities').API;
+var Move = require('app/entities').Move;
 var channel = Radio.channel('global');
 
 describe('Entities', function(){
     describe('API', function(){
         describe('initialization', function(){
-            it('registers an entities:spots request handler');
-            it('registers an entities:moves request handler');
-            it('registers an entities:move request handler');
-            it('registers an entities:move:reset request handler');
+            
+            it('registers an entities:spots request handler', function(){
+                var entitiesAPI = new EntitiesAPI();
+                var spots = "something";
+                this.replySpotsStub = sinon.stub(EntitiesAPI.prototype, 'replySpots', function(){ return spots });
+                var result = channel.request('entities:spots');
+                assert.isTrue(this.replySpotsStub.calledOnce, 'replySpots is called');
+                assert.equal(spots, result, 'the request handler returns the result of replySpots');
+                entitiesAPI.destroy();
+                this.replySpotsStub.restore();
+            });
+            it('registers an entities:moves request handler', function(){
+                var entitiesAPI = new EntitiesAPI();
+                var moves = "something";
+                this.replyMovesStub = sinon.stub(EntitiesAPI.prototype, 'replyMoves', function(){ return moves });
```

Comment:
Instead of assigning this to `this.replyMovesStub`, it would be better to assign it to a local variable, e.g. `var replyMovesStub = ...` This stub is only used within this single test and you don't want to accidentally make it available to other tests. The same goes with the other tests.